### ISSUE TITLE
Loss/Gain of sold stocks not displayed in holding

### ISSUE THREAD

**Issue by ianyu21** [afadil/wealthfolio#382]
Situation:
- I bought 100 shares of a stock at T1, T2, T3 respectively; cumulatively I have bought 300 shares averaging 10 dollars/share
- at T4, I sold the 200 shares of the stock at 8 dollars/share; I have 100 share remaining
- now at T5, the market price of the stock is rated at 9 dollar/share

The loss/gain of my account should be a sum of:
- Part A: the loss/gain realized from the sold 200 shares, which is 200 * (8-10) = -400 dollars
- Part B: the book loss/gain of the remaining 100 share, which is 100 * (9-10) = -100 dollars
So the total loss/gain is -500 dollars
However in the holdings detail table, only part B is shown, part A is completely missing.

Attaching an actual example:
photo 1: activities
photo 2 & 3: the holding info resulted from the activities

<img width="1149" height="607" alt="Image" src="https://github.com/user-attachments/assets/5d9199de-54fe-4939-a21c-5f53958cc4e0" />

<img width="1755" height="121" alt="Image" src="https://github.com/user-attachments/assets/c5d81928-b717-4d2f-ba12-23c59228fa57" />

<img width="1821" height="534" alt="Image" src="https://github.com/user-attachments/assets/ecf32c46-bada-48ab-8cbf-e67571b97ff6" />

**Comment by sticilface** [comment]
I'm also getting some strange behaviour.  
I've successfully sold some stock and it has updated the cash balance for one portfolio.  I did this using import for CSV. 
For another portfolio whenever I click on the next having selected all the import columns correctly... it just does nothing.  
Then I manually add the sell order, and it appears correctly.  

<img width="1515" height="81" alt="Image" src="https://github.com/user-attachments/assets/1a6e53b4-6e31-4db6-8a4e-637bf0502869" />

however, the amount sold for is not added to the CASH balance on the account and it now shows a loss = to sell amount. 

<img width="193" height="163" alt="Image" src="https://github.com/user-attachments/assets/9d41a3e2-6381-4597-a98a-9b8085e09348" />

**Comment by sticilface** [comment]
OK fixed it.  not sure why it cause the issue but I had currency set to GBp and not GBP which the original order was.  

> I'm also getting some strange behaviour. I've successfully sold some stock and it has updated the cash balance for one portfolio. I did this using import for CSV. For another portfolio whenever I click on the next having selected all the import columns correctly... it just does nothing. Then I manually add the sell order, and it appears correctly.
> 
> <img alt="Image" width="1515" height="81" src="https://private-user-images.githubusercontent.com/10061736/489735088-1a6e53b4-6e31-4db6-8a4e-637bf0502869.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgwMzg2MjQsIm5iZiI6MTc1ODAzODMyNCwicGF0aCI6Ii8xMDA2MTczNi80ODk3MzUwODgtMWE2ZTUzYjQtNmUzMS00ZGI2LThhNGUtNjM3YmYwNTAyODY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTE2VDE1NTg0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRjNWExNTM1YzlmMDNjYjI0ZDc0ZmI0MzY0ZjdiZjA0YjYxODQ0MjgzYTgxMDMxZTZlNTVkZjI0MmIxZTI4YmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Vf2ZtacEO4SG083Yoo7FfEbuG-sq8xWy0-QYxUyOYYM">
> however, the amount sold for is not added to the CASH balance on the account and it now shows a loss = to sell amount.
> 
> <img alt="Image" width="193" height="163" src="https://private-user-images.githubusercontent.com/10061736/489735763-9d41a3e2-6381-4597-a98a-9b8085e09348.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgwMzg2MjQsIm5iZiI6MTc1ODAzODMyNCwicGF0aCI6Ii8xMDA2MTczNi80ODk3MzU3NjMtOWQ0MWEzZTItNjM4MS00NTk3LWE5OGEtOWI4MDg1ZTA5MzQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTE2VDE1NTg0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM3MTVlZDAwOTlmY2I0YTM1YzAxODUyNGY3YWYwZmJkODE3ZTJiY2I1YzI5Mzk5NmM2MWI2YmRlODQwYjRmNjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.G-v-DPCzAWRY6LCQQLDq8U9c8bEF7kdoljmhs7uzpEw">

**Comment by afadil** [comment]
Notes:

-  We need to add Realized Gain/Loss for each lot.
-  We need a setting for the calculation method: FIFO, LIFO, AVG
